src/index.js

```diff
@@ -0,0 +1,23 @@
+// import lodash from 'lodash';
+import './style.css';
+
+const obj = {
+  description: 'Write an article',
+  completed: true,
+  index: 0,
+};
+const obj1 = {
+  description: 'Take a cofee',
+  completed: true,
+  index: 1,
+};
+
+const arr = [obj, obj1];
+
+// const _ = lodash;
+
+window.addEventListener('load', () => {
+  for (let i = 0; i < arr.length; i += 1) {
+    document.getElementById(arr[i].index).innerHTML = arr[i].description;
+  }
+});
```

Comment:
- [ ] As requested by the project requirements, you need to write a function that iterates over the `tasks` array and dynamically populates an HTML list item element for each task. However, your current code only works for two tasks. To enhance your code, please consider updating it to handle a variable number of tasks in the `tasks` list and kindly do it inside a function as a project requirement. 🙏  This will ensure that your code remains flexible and can accommodate any number of tasks provided. 👍